scratchpad/HWPF/HWPF.csproj

```diff
@@ -0,0 +1,296 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Project Sdk="Microsoft.NET.Sdk"><PropertyGroup><EnableDefaultCompileItems>false</EnableDefaultCompileItems><EnableDefaultContentItems>false</EnableDefaultContentItems><EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems></PropertyGroup><PropertyGroup><AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath></PropertyGroup>
+
+
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProductVersion>9.0.30729</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{F040162E-F6F3-4B09-809B-3095A07A6687}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>NPOI.HWPF</RootNamespace>
+    <AssemblyName>NPOI.ScratchPad.HWPF</AssemblyName>
+   <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
we need both  support `.net framework 4.6.1` and `.net standard 2.0` , not the `netcoreapp`